source/common/tracing/zipkin/zipkin_core_constants.cc

```diff
@@ -0,0 +1,35 @@
+#include "common/tracing/zipkin/zipkin_core_constants.h"
+
+namespace Zipkin {
+
+const std::string ZipkinCoreConstants::CLIENT_SEND = "cs";
+const std::string ZipkinCoreConstants::CLIENT_RECV = "cr";
+const std::string ZipkinCoreConstants::SERVER_SEND = "ss";
+const std::string ZipkinCoreConstants::SERVER_RECV = "sr";
+const std::string ZipkinCoreConstants::WIRE_SEND = "ws";
+const std::string ZipkinCoreConstants::WIRE_RECV = "wr";
+const std::string ZipkinCoreConstants::CLIENT_SEND_FRAGMENT = "csf";
```

Comment:
might be good to lazy declare these, since many don't use them at all and could be distracting. "ws" "wr" is only used in finagle on the client side (when the bytes hit the wire as opposed to when the client starts creating the request).. this feature could be punted unless you have a good hook for this.